src/main/java/FileReader.java

```diff
@@ -0,0 +1,17 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+class FileReader {
```

Comment:
Can you use Lombok for constructor generation?

Reply:
used